debian/rules

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/make -f
```

Comment:
Guess we should revert these changes, since we don't install the script as systemd user unit we don't need this.